velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -202,6 +203,8 @@ VeloxToSubstraitExprConvertor::toSubstraitLiteral(
           mapTypeKindToName(variantValue.kind()));
   }
 
+  literalExpr->set_nullable(variantValue.isNull());
```

Comment:
If value is null, then we probably should not execute the switch above which calls literalExpr->set_xxx. Would you add a test that uses null literals?

Reply:
if the value is null, it will call `toSubstraitNullLiteral`, the tests already contains null literals.

Reply:
Got it. Then, this code can simply use 'false':

```
literalExpr->set_nullable(false);
```

Reply:
Got it. Then, this code can simply use 'false':

```
literalExpr->set_nullable(false);
```

Reply:
> the tests already contains null literals by using nullEvery(2) when makeVectors.

I though that literals here come from expressions, e.g. c + 2 expression has literal '2'. Hence, we'll need to have an expression with null literal to test null literals, no? Something like f(c, null).

Reply:
sure, will add it.

Reply:
Hi @mbasmanova, I have updated it, please help take a review. Thanks.